Library/Formula/rswift.rb

```diff
@@ -0,0 +1,17 @@
+class Rswift < Formula
+  desc "Get strong typed, autocompleted resources like images, fonts and segues"
+  homepage "https://github.com/mac-cain13/R.swift"
+  url "https://github.com/mac-cain13/R.swift.git",
+    :tag => "v1.1.1",
+    :revision => "3a6db62164d8f50ccdf43e59894fffb672fd5e3f"
+  depends_on :xcode => "7.0"
+
+  def install
+    xcodebuild "-configuration", "Release", "-scheme", "rswift", "SYMROOT=symroot", "OBJROOT=objroot"
+    bin.install "symroot/Release/rswift"
+  end
+
+  test do
+    system "rswift", "-h"
```

Comment:
Sorry for the delay. If it’s too hard to create a resource for this it’s fine as is. Just replace `"rswift"` with `"#{bin}/rswift"`.